src/test/java/uk/gov/hmcts/sscs/personalisation/PersonalisationTest.java

```diff
@@ -85,30 +88,37 @@ public void customisePersonalisation() {
         assertEquals("http://link.com/GLSCRR", result.get(SUBMIT_EVIDENCE_LINK_LITERAL));
         assertEquals("http://link.com/progress/GLSCRR/expenses", result.get(CLAIMING_EXPENSES_LINK_LITERAL));
         assertEquals("http://link.com/progress/GLSCRR/abouthearing", result.get(HEARING_INFO_LINK_LITERAL));
+        assertNull(result.get(EVIDENCE_RECEIVED_DATE_LITERAL));
     }
 
     @Test
-    public void setAppealReceivedEventData() {
+    public void customisePersonalisationForEvidenceReceivedNotification() {
```

Comment:
test name does not follow a name convention such as  given/when/then or given/should